backends/candle/src/lib.rs

```diff
@@ -59,8 +59,9 @@ enum Config {
     NomicBert(NomicConfig),
     #[allow(dead_code)]
     Mistral(MistralConfig),
-    #[serde(rename = "new")]
     Gte(GTEConfig),
+    #[serde(rename = "new")]
+    GteAlibaba(GTEConfig),
```

Comment:
I think we can get away without modifying the config.

```rust
#[serde(alias = "new")]
Gte(GTEConfig),
```
Should be enough to support both model_type into the same config (since they are the same). That should help remove the need for the other code changes.